### DISCUSSION TITLE
Overclock questions for my 6600xt cards dual mining ETH-TON

### DISCUSSION THREAD

**Discussion by m2internet** [Lolliedieb/lolMiner-releases#1109] [General]
### Overclock questions for my 6600xt cards dual mining ETH-TON on lolMiner v1.43:

The lolMiner "Tuning guide Dual TON" shows either a suggested or absolute core MHz range for the 6600xt between 1100 - 1750.

**I'm pushing my 6600xt's pretty hard, well over the 1750 max in the ton guide:**

> 4 cards - 6600XT Samsung Memory - Core: 2100, VDD: 0, VDDCI: 825, MVDD: 1325, Memory: 1100
> hashrate: 31.14 MH on ETH, 1.277 GH on TON
> 2 cards - 6600XT Micron Memory - Core: 2100, VDD: 0, VDDCI: 825, MVDD:1325, Memory: 1065
> hashrate: 31.77 MH on ETH, 1.308 GH on TON
> Watts:  102w min TO 109w max, well below the TDP i found online of 160w (which I hope I'm right)
> Temp: 55C/70C

Considering the I'm well below the TDP, and not bad temps, how bad is it that I'm pushing those cores that high, for the cards lifespan?

Should I also not be going above the 1750 lolMiner tuning guide suggestion?

**Comment by m2internet** [comment]
anyone?  Was hoping that the lolminer devs could weigh in on those CORE clock ranges they had in their tuning guide. Thx :)

**Reply by Merkava83** [reply]
hello, 

i try this and it works 👍 

![image](https://user-images.githubusercontent.com/99438280/153488965-dbdc5e84-a32c-4311-808f-234f646ca119.png)
![image](https://user-images.githubusercontent.com/99438280/153489006-4db7192d-1464-456a-af11-008c42a62a97.png)

**Comment by elquijote1** [comment]
holas , anyone  know as is with team red mainer ? or is same result ? un saludo
